packages/infra/README.md

```diff
@@ -12,3 +12,88 @@ The `cdk.json` file tells the CDK Toolkit how to execute your app.
 - `cdk deploy` deploy this stack to your default AWS account/region
 - `cdk diff` compare deployed stack with current state
 - `cdk synth` emits the synthesized CloudFormation template
+
+## Creating a Certificate for IHE
+
+1. **Download Certificate and Private Key**
+
+   - Download your certificate (`ihe.staging.metriport.com.pem`) and private key (`PRIVATEKEY.key`) from their storage location, in this case, our Google Drive.
```

Comment:
Keep in mind people outside of Metriport might be using this. So:
- `('ihe.staging.metriport.com.pem')` could become `(for example, 'ihe.staging.metriport.com.pem')`
- remove `from their storage location, in this case, our Google Drive.`